pkg/cortex/cortex.go

```diff
@@ -221,7 +221,6 @@ func (t *Cortex) Run() error {
 
 // Stop gracefully stops a Cortex.
 func (t *Cortex) Stop() error {
-	t.server.Shutdown()
 	t.stop(t.target)
```

Comment:
I think you can just inline the implementation of `t.stop` now.

Reply:
I've done that - it hadn't occurred to me, but it is shorter.